wca-regulations.md

```diff
@@ -265,6 +265,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 9p) If an event has multiple rounds, then:
     - 9p1) At least 25% of competitors must be eliminated between consecutive rounds of the same event.
     - 9p2) The competitors who advance to the next round must be determined by ranking (best x competitors) or by result (all competitors with a better result than x) in the preceding round.
+        - 9p2a) For each round, advance conditions must be announced before the round starts, and should not be changed after it has begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
```

Comment:
**super nitpick and long explanation, sorry!**

I think "advancement conditions" sounds better to me "advance conditions".

Things get interesting if you Google ["advance vs advancement"](https://www.google.com/search?q=advance+vs+advancement&oq=advance+vs+advancement&aqs=chrome..69i57j0l3.4060j0j4&sourceid=chrome&ie=UTF-8), but here's my take on it as a native English speaker:

- advance can be both a noun and a verb, and when I see it, I think first think it's a verb.
- advancement can only be a noun

Given the above, when reading "advance conditions", my brain first parses "advance" as a verb, and then I get to "conditions" and the phrase makes no sense, so then I have to backtrack and interpret the phrase differently with advance as a noun. "advancement conditions" doesn't have this problem for me.

Reply:
Urg, on further thought, I just realized that in the phrase "advance conditions", "advance" is probably an adjective, not a noun... probably best not to listen to me on any grammar stuff, but I do maintain that "advancement" sounds nicer to me here.

Reply:
I agree with "advancement". "Advancing" could also work, but I prefer it less.

Reply:
Ok, I will change the PR tomorrow.
Thanks for the explanation @jfly!